README.md

```diff
@@ -107,6 +107,26 @@ provided by the devel module. Default values were used, except:
 As well as the database dump, the static files generated by the above process
 (user images, images embedded in content) have also been included.
 
+Drupal 8
+--------
+
+As above, aims to be realistic. Demo data is from the `devel_generate` module
```

Comment:
Pretty easily -- there is a self-explanatory set of links and a UI for the module. That said, maybe the right way is not to include a DB dump at all, but rather to use Drush commands (including the batch that `devel_generate` provides) and install/generate the site from scratch as part of the installer?